dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFileTest.java

```diff
@@ -30,12 +31,12 @@
 public class JsonResponseFileTest {
 
     // replace single quotes to double quotes to avoid \" messup.
-    private void check(String json, String... exp) throws JsonProcessingException {
+  private void check(String json, String... exp) throws IOException, JsonProcessingException {
```

Comment:
formatting

Reply:
In Jackson 2.9.10, `IOException` is thrown by [`ObjectMapper.readTree(String)`](https://source.corp.google.com/piper///depot/google3/third_party/java_src/jackson2/v2_9_10/jackson-databind-2.9.10.8/com/fasterxml/jackson/databind/ObjectMapper.java;bpv=1;bpt=1;rcl=478391678;l=2551?q=ObjectMapper.readTree&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental&gsn=readTree&gs=kythe%3A%2F%2Fgoogle3%3Flang%3Djava%3Fpath%3Dcom.fasterxml.jackson.databind.ObjectMapper%23ea6fe17ad8e74d2217ccfb651bd03123d02774db95d64767e405d380326fe4b3), which is why this exception had to be added.